erpnext/payroll/doctype/salary_detail/salary_detail.json

```diff
@@ -181,23 +186,12 @@
    "label": "Tax on additional salary",
    "read_only": 1
   },
-  {
-   "depends_on": "eval:doc.parenttype=='Salary Structure'",
-   "fieldname": "section_break_11",
-   "fieldtype": "Column Break"
-  },
-  {
-   "depends_on": "eval:doc.parenttype=='Salary Structure'",
-   "fieldname": "condition_and_formula_help",
-   "fieldtype": "HTML",
-   "label": "Condition and Formula Help",
```

Comment:
Why removed Condition and Formula Help?

Reply:
Because I have made the link for that which shows modal with every variable and formula examples.